api/models.py

```diff
@@ -28,22 +29,31 @@ def __str__(self):
 
 
 class Course(models.Model):
+    def __str__(self):
+        return self.num
+
     num = models.IntegerField(default=0, primary_key=True)
     name = models.CharField(max_length=200)
     credit = models.IntegerField(default=0)
     prof_name = models.ForeignKey(Professor, on_delete=models.CASCADE)
     classroom = models.CharField(max_length=200)
-    weekday = models.CharField(max_length=100)  # 강의 요일
-    start_time = models.TimeField(auto_now=False, auto_now_add=False)  # 강의시간
-    finish_time = models.TimeField(auto_now=False, auto_now_add=False)
+    weekday = models.CharField(max_length=100, default='')  # 강의 요일
+    start_time = models.TimeField(auto_now=False, auto_now_add=False, null=True)  # 강의시간
+    finish_time = models.TimeField(auto_now=False, auto_now_add=False, null=True)
 
 
 class Basket(models.Model):
+    def __str__(self):
+        return self.course_num
+
     course_num = models.ForeignKey(Course, on_delete=models.CASCADE)
     std_id = models.ForeignKey(Student, on_delete=models.CASCADE)
 
 
 class Registration(models.Model):
+    def __str__(self):
+        return self.course_num
+
     course_num = models.ForeignKey(Course, on_delete=models.CASCADE, related_name="reg_crs")
```

Comment:
여기도 위에서 드린 피드백과 마찬가지로 필드명을 `course`로 바꾸는 게 어떨까 싶습니다🐵
related_name의 경우, entity 간 relation을 나타내 주려고 하신 것 같아요!(아니라면 죄송해요😅) 
하지만 django models의 related_name은 그 개념과는 조금 다릅니다. django orm에서 인스턴스를 단위로, 즉 객체 단위로, 해당 객체에 연결된 다른 객체 또는 객체들을 참조하기 위한 변수를 설정해주시는 거라고 보시면 될 것 같아요!
registration의 course와 같은 경우, 두 모델이 foreign key로 연결되었기 때문에 course 필드의 `related_name`은 하나의 course 객체와 연결된 여러 개의 registration 객체를 불러오는 변수이므로, `related_name="registrations"` 같은 이름이 적절할 것 같아요🤩

```suggestion
    course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name='registrations')
```